src/components/Autocomplete/__snapshots__/Autocomplete.spec.jsx.snap

```diff
@@ -2,6 +2,18 @@
 
 exports[`Autocomplete [common] example testing should match snapshot(s) for 1.basic 1`] = `
 <DropMenu
+  ContextMenu={
```

Comment:
You're going to see a lot of these snapshot changes. I added the ability to pass props through `DropMenu` down into `ContextMenu`.